From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 11 Apr 2014 09:28:55 +0100
Subject: configure.ac include acscrypt.m4

acscrypt.m4 contains the macro CHECK_SYSCTL_HW_USERMEM.  This should
be put in check_sysctl_hw_usermem.m4 so it can be found automatically,
but that would be more appropriate to do upstream, and this is a
minimal patch.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index b9445a7..8f73c42 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,6 +2,7 @@ AC_INIT([scrypt],[1.1.6])
 AC_CONFIG_AUX_DIR([config.aux])
 AM_INIT_AUTOMAKE()
 AM_MAINTAINER_MODE
+m4_include([acscrypt.m4])
 AC_PROG_CC
 
 # Check for clock_gettime.  On some systems, this is provided via librt.
